tests/InstrEngineTests/NaglerInstrumentationMethod/NaglerInstrumentationMethod.cpp

```diff
@@ -1142,40 +1158,30 @@ HRESULT CInstrumentationMethod::GetType(IModuleInfo* pModuleInfo, const CLocalTy
     return hr;
 }
 
-
 HRESULT CInstrumentationMethod::AllowInlineSite(_In_ IMethodInfo* pMethodInfoInlinee, _In_ IMethodInfo* pMethodInfoCaller, _Out_ BOOL* pbAllowInline)
 {
     *pbAllowInline = FALSE;
     return S_OK;
 }
 
-
 HRESULT CInstrumentationMethod::ExceptionCatcherEnter(
     _In_ IMethodInfo* pMethodInfo,
     _In_ UINT_PTR   objectId
     )
 {
-    CComPtr<IProfilerManagerLogging> spLogger;
```

Comment:
Consider reverting these type of changes. I think it's more important to have it test the IProfilerManager::GetLoggingInstance method and make sure that your new logging changes still work over time.

Reply:
The problem is that now that method prepends the instrumentation method guid and breaks the test verification which parses XML. I am QI for ProfilerManager4 and invoke GetGlobalLogger to work around this.

---
In reply to: [341220369](https://github.com/microsoft/CLRInstrumentationEngine/pull/146#discussion_r341220369) [](ancestors = 341220369)

Reply:
I'll try to update the tests in the next PR to ignore these fields when parsing XML

---
In reply to: [341282058](https://github.com/microsoft/CLRInstrumentationEngine/pull/146#discussion_r341282058) [](ancestors = 341282058,341220369)